### PR TITLE
Add examples to README

### PR DESCRIPTION
**Pull request by zenangst**
This PR adds a description and some code examples to show how `Imaginary` is used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,47 @@
...
+let imageView: UIImageView()
+let imageURL: NSURL(string: "https://avatars2.githubusercontent.com/u/1340892?v=3&s=200")
+imageView.setImage(URL: imageURL)
+```
+
+### How to apply placeholder images
+```swift
+let imageView: UIImageView()
+let placeholder = UIImage(named: "PlaceholderImage")
+let imageURL: NSURL(string: "https://avatars2.githubusercontent.com/u/1340892?v=3&s=200")
```
Comment (onmyway133) [inline README.md:31]: You mean `imageURL = URL(string: ...)` ?

File: README.md [hunk 2]
```diff
@@ -8,10 +8,47 @@
...
+This process should be easy, straight-forward and hassle free, and with `Imaginary`, it is.
+With a narrow yet flexible public API, you can set images using URL's, add
+placeholders, pre-process images and use callbacks for when images are set.
+
 ## Usage
 
+### Set image with URL
+```swift
+let imageView: UIImageView()
+let imageURL: NSURL(string: "https://avatars2.githubusercontent.com/u/1340892?v=3&s=200")
```
Comment (onmyway133) [inline README.md:23]: I think we will go with `URL` 😄

File: README.md (hunk 1 above)
Comment (zenangst) [reply README.md:31]: This is for the `2.2` version.

File: README.md (hunk 2 above)
Comment (zenangst) [reply README.md:23]: See my previous comment.

**Pull request merged**